browser/src/control/jsdialog/Widget.DrawingArea.js

```diff
@@ -40,10 +40,15 @@ function _drawingAreaControl (parentContainer, data, builder) {
 	image.tabIndex = 0;
 	image.draggable = false;
 	image.ondragstart = function() { return false; };
-	image.setAttribute('data-cooltip', data.text);
 
-	if (builder.map) {
-		L.control.attachTooltipEventListener(image, builder.map);
+	if (data.text) {
+		image.setAttribute('data-cooltip', data.text);
+
+		if (builder.map) {
+			L.control.attachTooltipEventListener(image, builder.map);
+		}
+	} else if (data.aria && data.aria.label) {
+		image.setAttribute('aria-label', data.aria.label);
```

Comment:
when `aria.label` is set in core?

I was wondering if it should be the priority and only when missing use `text` tooltip?

Reply:
So this will not be no longer needed for this particular widget (layouts in impress) because Parth converts it to the Icon View (https://github.com/CollaboraOnline/online/pull/11990)

But question remains :) in general what do we prefer, let me approve this as it is now minimal change as you said. Thanks